### PR TITLE
feat(wedof): add new actions

### PR DESCRIPTION
**Pull request by obenazouz-wedof**
## What does this PR do?

Fixes # (issue)

### CODE REVIEW

File: packages/ui/common/src/lib/environments/environment.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 export const environment = {
   production: false,
-  apiUrl: 'http://localhost:3000/v1',
+  apiUrl:
```
Comment (kishanprmr) [inline packages/ui/common/src/lib/environments/environment.ts]: @obenazouz-wedof can you please revert these changes in this file?

Comment (obenazouz-wedof) [reply packages/ui/common/src/lib/environments/environment.ts]: done :) ! thank you that was a little mistake

File: packages/server/api/.env [hunk 2]
```diff
@@ -15,4 +15,4 @@ AP_TELEMETRY_ENABLED=false
 AP_TRIGGER_DEFAULT_POLL_INTERVAL=1
 AP_WEBHOOK_TIMEOUT_SECONDS=30
 AP_PIECES_SYNC_MODE=NONE
-AP_WEBHOOK_URL="http://localhost:3000"
```
Comment (abuaboud) [inline packages/server/api/.env]: Can you please reverse the whitespace there, We have hooks to forbid commit .env and this will cause an issue for other when they are merging.

Comment (obenazouz-wedof) [reply packages/server/api/.env]: done 👍 , thnx

**Review by abuaboud — APPROVED** [review]
